work/v1/types.go

```diff
@@ -233,6 +233,8 @@ type ManifestWorkList struct {
 // deployed in the managed cluster.
 // When the agent is removed from managed cluster, cluster-admin on managed cluster
 // can delete appliedmanifestwork to remove resources deployed by the agent.
+// The namee of the appliedmanifestwork must be in the format of
+// {hash of hub's first kube-apiserver url}-{manifestwork name}
```

Comment:
typo: namee :)

Reply:
The comment raises an interesting question I hadn't thought of before... does this effectively mean that the first kube-apiserver URL must be immutable?

Reply:
good point, what we want is a unique immutable ID to identify hub. For OCP,  maybe we can also use the cluster id of hub cluster?